src/components/pages/doc.less

```diff
@@ -0,0 +1,17 @@
+@doc-page-control-panels-column-width: @edu-control-height-sm + (2 * @edu-content-padding);
+
+.DocPage {}
+
+.DocPage-controlPanels {
+  position: sticky;
+  bottom: 0;
+  margin-right: 50px;
+  display: grid;
+  grid-gap: 5px;
```

Comment:
`grid-gap` is replaced by `gap` (as it can now also be used for flex layout :imp:), so let's go with the new recommended rule (we have Autoprefixer anyways for older browser quirks).

Reply:
Addendum: I'm currently changing all `grid-gap` to `gap` anyways.

Reply:
Too late, I also changed it 😶‍🌫️ Hope it doesn't conflict with your changes!

Reply:
No worries, wer zu spät kommt, den bestraft das Leben (Michail Gorbatschow, allegedly)